test/with_dummyserver/test_https.py

```diff
@@ -66,6 +68,30 @@ def test_set_ssl_version_to_tlsv1(self):
         r = self._pool.request('GET', '/')
         self.assertEqual(r.status, 200, r.data)
 
+    def test_client_intermediate(self):
+        client_cert, client_key = DEFAULT_CLIENT_CERTS['certfile'], \
+                                  DEFAULT_CLIENT_CERTS['keyfile']
+        serial = b'16180339887498948482045868343656381177203091798' + \
+                 b'05762862135448622705260\\'
```

Comment:
It might be better to put this serial number in the `DEFAULT_CLIENT_CERTS` dictionary, so that it's proximate to the filename and so anyone changing that file will see it.